doc/reference/tooling/tt_cli/pack.rst

```diff
@@ -36,6 +36,46 @@ The package contains the following files:
 
 You can also pass various :ref:`options <tt-pack-options>` to the ``tt pack`` command to adjust generation properties, for example, customize a bundle name, choose which artifacts should be included, specify the required application dependencies.
 
+.. _tt-pack-systemd:
+
+systemd unit parameters
+-----------------------
+
+You can customize your application's systemd unit file that is generated by ``tt pack``.
+To add parameters to the unit file, define them in a YAML file named ``systemd-unit-params.yml``
+in the application directory.
```

Comment:
> in the application directory.

It would be convenient to have a link for `application directory`, so a reader can understand where this directory is placed. Not sure which document is the best to refer to (maybe, https://www.tarantool.io/en/doc/latest/book/admin/instance_config/#creating-and-developing-an-application).